packages/internals/src/get-generators/generatorResolvers/prisma-client-js/prismaClientResolver.ts

```diff
@@ -78,10 +78,10 @@ export async function prismaClientResolver(baseDir: string, version?: string) {
     }
 
     if (!prismaCliDir) {
-      await runPackageCmd(projectRoot, 'add', `prisma@${version ?? 'latest'}`, '-D')
+      await runPackageCmd(projectRoot, 'add', `prisma@${version ?? 'latest'}`, '-D', '--silent')
```

Comment:
I think we can completely silence npm output here (`stdio: 'ignore'` option for `execa.command` in `runPackageCmd`)

Reply:
That also then omits the little loading bar, right?